cmd/generate/runner.go

```diff
@@ -33,5 +35,36 @@ func (r *runner) Run(cmd *cobra.Command, args []string) error {
 }
 
 func (r *runner) run(ctx context.Context, cmd *cobra.Command, args []string) error {
+	fmt.Fprintf(r.stdout, "Creating vault client using opsctl\n")
+
+	vaultClient, err := creaeteVaultClientUsingOpsctl(ctx, r.flag.GitHubToken, r.flag.Installation)
+	if err != nil {
+		return microerror.Mask(err)
+	}
+
+	var dec *decrypter.Decrypter
+	{
+		c := decrypter.Config{
+			VaultClient: vaultClient,
+		}
+
+		dec, err = decrypter.New(c)
+		if err != nil {
+			return microerror.Mask(err)
+		}
+
+	}
+
+	if len(args) != 1 {
+		fmt.Fprintf(r.stderr, "Error: Expected the first argument to encrypted blob")
+	}
+
+	decrypted, err := dec.Decrypt(ctx, []byte(args[0]))
+	if err != nil {
+		return microerror.Mask(err)
+	}
+
+	fmt.Fprintf(r.stdout, "Decrypted: %s\n", decrypted)
```

Comment:
This part should be removed eventually. It's here to test things work.

```
$ export CONFIG_CONTROLLER_GITHUB_TOKEN="..."
```

Successful run:

```
$ go run . generate --app does-not-matter --config-version does-not-matter --installation gauss 'vault:v1:p8FdQPSs8R16X6A/NSi0KmJa+w7ZNr54vJnpxNDz/q1/Eq97fqegBwyZYCtid993'
Creating vault client using opsctl
Decrypted: decrypted-test-value
```

When something goes wrong:

```
$ go run . generate --app does-not-matter --config-version does-not-matter --installation gauss 'abc'
Creating vault client using opsctl
Error: Error making API request.

URL: POST https://127.0.0.1:58388/v1/transit/decrypt/config
Code: 400. Errors:

* invalid ciphertext: no prefix
        /Users/kopiczko/go/src/github.com/giantswarm/config-controller/pkg/decrypter/decrypter.go:70
        /Users/kopiczko/go/src/github.com/giantswarm/config-controller/pkg/decrypter/decrypter.go:50
        /Users/kopiczko/go/src/github.com/giantswarm/config-controller/cmd/generate/runner.go:64
        /Users/kopiczko/go/src/github.com/giantswarm/config-controller/cmd/generate/runner.go:31
        /Users/kopiczko/go/src/github.com/giantswarm/config-controller/main.go:136
        /Users/kopiczko/go/src/github.com/giantswarm/config-controller/main.go:29
exit status 1
```